released/packages/coq-simple-io/coq-simple-io.1.7.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "Li-yao Xia <lysxia@gmail.com>"
+authors: [ "Li-yao Xia" "Yishuai Li" ]
+homepage: "https://github.com/Lysxia/coq-simple-io"
+bug-reports: "https://github.com/Lysxia/coq-simple-io/issues"
+license: "MIT"
+dev-repo: "git+https://github.com/Lysxia/coq-simple-io.git"
+build: [make "build"]
+run-test: [make "test"]
+install: [make "install"]
+depends: [
+  "ocaml"
+  "coq" {>= "8.11" & < "8.15~"}
```

Comment:
```suggestion
  "coq" {>= "8.11" & < "8.16~"}
```
@Lysxia I tried locally, and the package works fine on Coq 8.15.0. Is there some reason not to allow 8.15?

Reply:
No reason :) I just forgot 8.15 is the latest version.

Reply:
Thanks. Note that it's allowed to submit packages with no upper bound, e.g., `"coq" {>= "8.11"}`. We will then set a bound if we notice it breaks.

Reply:
I see! I'm still partial to conservative upper bounds if that's okay.